notes/eligibility-summary.vbs

```diff
@@ -23444,6 +23444,82 @@ class stat_detail
 		stat_mont_form_recvd_date = replace(stat_mont_form_recvd_date, " ", "/")
 		If stat_mont_form_recvd_date = "__/__/__" Then stat_mont_form_recvd_date = ""
 
+		'Herehere
+		'TODO: Is this okay placement? 
```

Comment:
This is good placement for reading the information but the interaction should come after all the gathering of information. The interruption of the script operation shouldn't happening within the class because we likely will move this class to being available to any script. However, this means you will need a way to output the information about the review code and have it be a class parameter.